recipes/kmer-db/build.sh

```
@@ -1,9 +1,11 @@
 #!/bin/bash
 
+uname_S=`uname -s 2>/dev/null || echo not`
+if [ "$uname_S" == "Darwin" ]; then brew install gcc@12;
```

Shellcheck

[error] Couldn't find 'fi' for this 'if'.

(SC1046)